### ISSUE TITLE
[upgrade] must remove kmod-drbd before starting the yum update

### ISSUE THREAD

**Issue by fabbione** [ClusterLabs/anvil#377]
from PR-376 the update fails to install the new version kmod-drbd because the old one is still installed.

**Comment by fabbione** [comment]
CI update fakes an update from:

kmod-drbd-9.2.4-1.el8.x86_64 to kmod-drbd-9.2.4-99.el8.x86_64

after the update tool has gone around updating:

# rpm -qa|grep kmod-drbd
kmod-drbd-9.2.4-99.el8.x86_64
kmod-drbd-4.18.0-500.el8-9.2.4-1.el8.x86_64
akmod-drbd-9.2.4-99.el8.x86_64

the old kmod-drbd is still there.

akmod can´t install the new kmod because the old one is still present.

**Comment by fabbione** [comment]
duplicate of https://github.com/ClusterLabs/anvil/issues/349
